resource/ui/templates/monitor.html

```diff
@@ -0,0 +1,21 @@
+<!--
+Copyright 2014 The Cockroach Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See the AUTHORS file
+for names of contributors.
```

Comment:
I'd prefer if people keep the author tag. I find these invaluable because git blame quickly attributes sweeping changes to the wrong authors due to refactoring. People need to "own" files. This IMO was a huge problem at Other Companies We've Worked At (tm)